tests/unit/ecs_tasks/test_s3.py

```diff
@@ -725,3 +730,46 @@ def test_it_caches_manifests(mock_fetch):
         [call("s3://path/to/manifest1.json"), call("s3://path/to/manifest2.json")],
         any_order=True,
     )
+
+
+def test_s3transfer_locked_version():
+    """
+    https://github.com/boto/s3transfer/issues/82#issuecomment-837971614
+
+    We have a monkey patch in place to allow us using boto3's upload_fileobj
+    when we write back to S3. The issue is that while the method offers a nice
+    wrapper around file operations, such as implementing multipart upload only
+    when needed, we don't get the VersionId back as we would by using
+    put_object. This monkey patch is in place while we wait some pull requests
+    to be merged. In the meanwhile, here is a test that allow us to notice
+    any change on the version we use on s3transfer, in order to add extra
+    protection against automated library upgrade PRs that may silently introduce
+    issues.
+    """
+    assert s3transfer.__version__ == "0.6.0"
```

Comment:
I discussed this with @HieronymusLex - may seem a bit over cautious but I think it's worth it given Dependabot may make very easy to silently upgrade and leave dangerous things unnoticed.